cypress/support/bodyTestHelper.js

```diff
@@ -153,29 +153,33 @@ export const visibleImageWithCaption = figure => {
   figure.should('to.have.descendants', 'figcaption');
 };
 
-export const errorMessage = service => {
+export const errorMessage = serviceConfig => {
```

Comment:
These three helpers are only used in one file now. I don't think they should be helpers because of this. I'd move them all into errorPageArticle.

NB we shouldn't use const, but removing consts can be done in another PR

Reply:
Have moved the helpers all into errorPageArticle.

Reply:
Have moved the helper functions into errorPageArticle.